### PR TITLE
Enable nullable reference types for Jellyfin.Server

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Jellyfin.Server/Program.cs [hunk 1]
```diff
@@ -86,6 +88,12 @@ private static async Task StartApp(StartupOptions options)
         {
             var stopWatch = new Stopwatch();
             stopWatch.Start();
+
+            // Log all uncaught exception to std error
+            static void UnhandledExceptionToConsole(object sender, UnhandledExceptionEventArgs e) =>
```
Comment (dkanada) [inline Jellyfin.Server/Program.cs]: Isn't the lambda supposed to go on the next line?

Comment (Bond-009) [reply Jellyfin.Server/Program.cs]: It's not a lambda, it's an local function.

File: Jellyfin.Server/Program.cs [hunk 2]
```diff
@@ -129,7 +139,7 @@ private static async Task StartApp(StartupOptions options)
 
             _logger.LogInformation(
                 "Jellyfin version: {Version}",
-                Assembly.GetEntryAssembly().GetName().Version.ToString(3));
+                Assembly.GetEntryAssembly()!.GetName().Version!.ToString(3));
```
Comment (JustAMan) [inline Jellyfin.Server/Program.cs]: what does this `!` do?

**Comment by Bond-009** [comment]
@JustAMan they were by default, by enabling nullable reference types you have to explicitly make them nullable.
Ref: https://devblogs.microsoft.com/dotnet/nullable-reference-types-in-csharp/

File: Jellyfin.Server/Program.cs (hunk 2 above)
Comment (cvium) [reply Jellyfin.Server/Program.cs]: > The trailing ! on an expression tells the compiler that, despite what it thinks, it shouldn’t worry about that expression being null.

**Pull request merged**